packages/workbox-webpack-plugin/index.js

```diff
@@ -62,6 +62,25 @@ class WorkboxBuildWebpackPlugin {
       config.globPatterns = ['**/*.{html,js,css}'];
     }
 
+    if (compilation.options.output.publicPath && !config.manifestTransforms) {
+      const publicPath = compilation.options.output.publicPath;
```

Comment:
Hah. Nothing wrong with that. I'm fine with us switching this over to destructuring.